.github/workflows/wheel_win_x86_64.yml

```diff
@@ -14,7 +14,7 @@ env:
 
   # Python build settings
   CIBW_BEFORE_BUILD: |
-    pip install pybind11 ninja cmake
+    pip install pybind11 ninja cmake && powershell.exe New-ItemProperty -Path "HKLM:\\SYSTEM\\CurrentControlSet\\Control\\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
```

Comment:
I'll keep working on this in #293  and merge onto this PR when ready,

Reply:
Sure! Thank you, Lee!